File: singup/app/src/main/java/com/example/singup/MainActivity.java
package com.example.singup;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.text.Editable;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.RadioButton;
import android.widget.RadioGroup;

public class MainActivity extends AppCompatActivity {

    RadioGroup rGoup;
    String Gender;
    RadioButton male,female;
    EditText profileName, emailID;
    Button submit_id;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        rGoup=findViewById(R.id.gender);
        profileName= findViewById(R.id.profileName);
        emailID= findViewById(R.id.emailID);
        submit_id=findViewById(R.id.submit_id);

        submit_id.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                if(checkAllComplete())
                    System.out.println("Imara");
            }
        });


    }

    public boolean checkAllComplete(){
        Editable pName=profileName.getText();
        Editable email=emailID.getText();
        int selectedGNumber=rGoup.getCheckedRadioButtonId();
        if(selectedGNumber==1)
            Gender="Male";
        else if(selectedGNumber==2)
            Gender="Female";

        return (pName!=null && email!=null && email!=null && Gender!=null);
    }
}

File: RecylerViewMovies/app/src/main/java/com/example/recylerviewmovies/MainActivity.kt
package com.example.recylerviewmovies

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.RecyclerView
import com.example.recylerviewmovies.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var value :Project
    private lateinit var list:ArrayList<Project>
    private lateinit var list_Recyclerview:RecyclerView
    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        list_Recyclerview= binding.recyclerViewProjects
            value =Project(
            "First view",
            "this movies is cool", 1
        );
        list.add(value)


    }
}